src/main/java/com/onarandombox/MultiverseCore/commands/DumpsCommand.java

```diff
@@ -0,0 +1,242 @@
+    private final MultiverseCore plugin;
+    private final MVWorldManager worldManager;
+
+    @Inject
+    public DumpsCommand(@NotNull MVCommandManager commandManager,
+                        @NotNull MultiverseCore plugin,
+                        @NotNull MVCoreConfig config,
+                        @NotNull MVWorldManager worldManager) {
+        super(commandManager);
+        this.config = config;
+        this.plugin = plugin;
+        this.worldManager = worldManager;
+
+        registerFlagGroup(CommandFlagGroup.builder("mvdumps")
+                .add(CommandFlag.builder("--pastebincom")
+                        .addAlias("-b")
+                        .build())
+                .add(CommandFlag.builder("--github")
+                        .addAlias("-g")
+                        .build())
+                .add(CommandFlag.builder("--hastebin")
+                        .addAlias("-h")
+                        .build())
+                .add(CommandFlag.builder("--pastegg")
+                        .addAlias("-p")
+                        .build())
+                .add(CommandFlag.builder("--logs")
+                        .addAlias("-l")
+                        .build())
+                .add(CommandFlag.builder("--include-plugin-list")
+                        .addAlias("-l")
+                        .build())
+                .build());
+    }
+
+    @Subcommand("dumps")
+    @CommandPermission("multiverse.core.dumps")
+    @CommandCompletion("@flags:groupName=mvdumps")
+    @Syntax("--pastebincom --hastebin --pastegg --logs --include-plugin-list")
+    @Description("{@@mv-core.dumps.description}")
+    public void onDumpsCommand(CommandIssuer issuer,
+
+                               @Optional
+                               @Syntax("--pastebincom --hastebin --pastegg --logs --include-plugin-list")
+                               @Description("{@@mv-core.dumps.flags.description}")
+                               String[] flags
+    ) {
+        ParsedCommandFlags parsedFlags = parseFlags(flags);
+
+        MVVersionEvent versionEvent = new MVVersionEvent();
+
+        this.addDebugInfoToEvent(versionEvent);
+        plugin.getServer().getPluginManager().callEvent(versionEvent);
+
+        final String versionInfo = versionEvent.getVersionInfo();
+
+        if (parsedFlags.hasFlag("--include-plugin-list")) {
+            versionEvent.putDetailedVersionInfo("plugins.md", "# Plugins\n\n" + getPluginList());
+        }
+
+        final Map<String, String> files = versionEvent.getDetailedVersionInfo();
+
+
+        BukkitRunnable logPoster = new BukkitRunnable() {
+            @Override
+            public void run() {
+                HashMap<String, String> pasteURLs = new HashMap<>();
+                boolean hasArgs = false;
+
+                if (parsedFlags.hasFlag("--pastebincom")) {
+                    hasArgs = true;
+                    sendMessage(issuer, "Uploading debug info to https://pastebin.com");
```

Comment:
Prob good idea to make all the sendMessage string into i18n instead

Reply:
Adding to the todo